tests/regionizers/test_administrative_boundary_regionizer.py

```diff
@@ -126,8 +171,9 @@ def test_empty_region_full_bounding_box(toposimplify: Union[bool, float]) -> Non
         (0),
     ],
 )
-def test_no_empty_region_full_bounding_box(toposimplify: Union[bool, float]) -> None:
+def test_no_empty_region_full_bounding_box(toposimplify: Union[bool, float], request: Any) -> None:
```

Comment:
extract test and prepare real data for toposimplify parameter

Reply:
Extracted and this is the only test which performs an overpass query